torch/csrc/jit/passes/quantization.cpp

```diff
@@ -12,6 +12,17 @@ namespace jit {
 namespace {
 // QuantizerUtils
 
+void insertNodeHelper(Node* new_n, Node* ins_node, bool insert_after) {
```

Comment:
it's not a very useful function imho :)

Reply:
There is `WithInsertPoint` class for this - please don't reimplement it.